### DISCUSSION TITLE
Feed-wise refresh rate

### DISCUSSION THREAD

**Discussion by NTL-Remi** [spacecowboy/Feeder#295] [Ideas]
Hello, thanks for the app !

Would you consider adding a per-feed syncing frequency setting (with i'd guess the current global one as a default for new additions) ?
What sparks my request is a [website](https://cheapskatesguide.org/policy-on-robots.html) I just stumbled upon, that warns RSS and crawler users about being banned if putting too much pressure on their limited server setup's bandwidth.
I also use Feeder to subscribe to my city's transport system alerts,
and stepping back, it makes sense to me that one doesn't need the same response time for a real-time alert system and a monthly blog.

I have zero experience with Android environment, so I have no clue how this could interfere with battery optimization options, scheduling or whatever.

As I write these lines, I realize it might also be worth considering to have a wall clock time instead of a frequency, to get the notifications at a wanted time, for lunch for example.

**Comment by spacecowboy** [comment]
> Would you consider adding a per-feed syncing frequency setting (with i'd guess the current global one as a default for new additions) ?

No.

> What sparks my request is a [website](https://cheapskatesguide.org/policy-on-robots.html) I just stumbled upon, that warns RSS and crawler users about being banned if putting too much pressure on their limited server setup's bandwidth.

They should return proper CacheHeaders in their response. Feeder will respect those and make sure not to call the server too often, instead the cached response will be used or it will be validated against the server to be still valid. This is the canonical way to handle this.

> As I write these lines, I realize it might also be worth considering to have a wall clock time instead of a frequency, to get the notifications at a wanted time, for lunch for example.

Essentially not supported by modern Android.
